docs/guide/custom-functions.md

```diff
@@ -140,16 +220,72 @@ the calculations.
 // arguments here are displayed just to show the structure
 public hyper(ast, state) {
     return 'Hyperformula'.length;
-  }
-};
+  };
+```
+
+### `runFunction()`
+
+To test your function's configuration, run the built-in `runFunction()` function.
+
+The `runFunction()` function:
+- Validates your function's [optional parameter](#optional-parameters) settings
+- Validates your [argument validation options](#argument-validation-options)
+- Checks if values returned by your function are in the right format
+
+```javascript
+import { FunctionPlugin } from 'hyperformula';
+
+export class CountHF extends FunctionPlugin {
+
+  public static implementedFunctions = {
+    'HYPER': {
+      method: 'hyper',
+      // set your optional parameters
+      arrayFunction: false,
+      // set your argument validation options
+      parameters: {
+        passSubtype: false
+      }
+    }
+  };
+
+  public hyper(ast, state) {
+    this.runFunction(HYPER())
+    
+    return 'Hyperformula'.length
+    }
+  };
+```
+
+### Throwing errors
```

Comment:
We're not throwing errors here. We're returning them. And they are correctly handled and displayed with translation.

Reply:
@wojciechczerniak - done: 304dfa0